platforms/common/include/px4_platform_common/posix.h

```diff
@@ -114,6 +114,11 @@ __END_DECLS
 #error "No TARGET OS Provided"
 #endif
 
+#if defined(__PX4_QURT)
```

Comment:
Can you merge this into the ifdefs above where we already check if PX4_NUTTX, PX4_POSIX, etc (and error if no target OS provided).

Reply:
Yep! Will do that asap.